modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryMvTableStorageTest.java

```diff
@@ -57,12 +70,18 @@ void setUp(
                     "mock.tables.foo{ partitions = 512, dataStorage.name = " + VolatilePageMemoryStorageEngine.ENGINE_NAME + "}"
             )
             TablesConfiguration tablesConfig
-    ) {
+    ) throws Exception {
         var ioRegistry = new PageIoRegistry();
 
         ioRegistry.loadFromServiceLoader();
 
         initialize(new VolatilePageMemoryStorageEngine("node", engineConfig, ioRegistry, pageEvictionTracker), tablesConfig);
+
+        // Track how many times a page was forgotten (when emptying a Data Page). We need it to be able to poll while
+        // awaiting till a page finally gets freed (as MV and index data destruction is async). We don't use verify()
+        // for this end because it throws an AssertionError if the expectation is not satisfied, which is not convenient.
```

Comment:
Why is it "verify" not convenient?

Reply:
Switched to `verify(..., timeout())`, as suggested, thanks!